custom_components/tuya_local/devices/blitzwolf_bw_lt31.yaml

```diff
@@ -0,0 +1,155 @@
+name: Blitzwolf BW-LT31 LED Strip
```

Comment:
Top level name should be just LED strip, without branding.  The branding belongs under products.